src/Native/Runtime/arm/ExceptionHandling.S

```diff
@@ -4,4 +4,295 @@
 
 #include <unixasmmacros.inc>
 
+.syntax unified
+.thumb
+
 // TODO: Implement Arm support
+
+//////////////////////////////////////////////////////////////////////////////////////////////////////////////
+//
+// RhpThrowHwEx
+//
+// INPUT:  RDI:  exception code of fault
+//         RSI:  faulting RIP
+//
+// OUTPUT:
+//
+//////////////////////////////////////////////////////////////////////////////////////////////////////////////
+NESTED_ENTRY RhpThrowHwEx, _TEXT, NoHandler
+
+#ifdef _DEBUG
+	bl C_FUNC(NYI_Assert)
+#endif
+
+  // struct PAL_LIMITED_CONTEXT
+  // {
+  // }
+
+  // allocate outgoing args area and space for the ExInfo
+
+  // init the exception object to null
+  // init to the first pass
+  // ExKind.HardwareFault
+
+  // link the ExInfo into thread's ExInfo chain
+  // pExInfo->m_pPrevExInfo = m_pExInfoStackHead
+  // m_pExInfoStackHead = pExInfo
+
+  // set the exception context field on the ExInfo
+  // rdx <- PAL_LIMITED_CONTEXT*
```

Comment:
The comments that reference AMD64 registers are throw away.

It would be better to either not copy these comments at all; or copy them from the Windows ARM .asm files.